pkgs/by-name/va/vaultwarden/webvault.nix

```diff
@@ -2,22 +2,28 @@
   lib,
   buildNpmPackage,
   fetchFromGitHub,
+  fetchurl,
   git,
   nixosTests,
   python3,
   vaultwarden,
 }:
 
 let
-  version = "2025.1.1";
+  version = "2025.5.0";
 
   suffix = lib.head (lib.match "[0-9.]*([a-z]*)" version);
 
   bw_web_builds = fetchFromGitHub {
     owner = "dani-garcia";
     repo = "bw_web_builds";
```

Comment:
Why don't we build from https://github.com/vaultwarden/vw_web_builds/tree/v2025.5.0 instead and skip the need to apply patches?